clap_builder/src/builder/command.rs

```diff
@@ -1039,6 +1039,14 @@ impl Command {
 
         Usage::new(self).create_usage_with_title(&[])
     }
+
+    /// Extend [`Command`] with [`CommandExt`] data
+    #[cfg(feature = "unstable-ext")]
+    #[allow(clippy::should_implement_trait)]
+    pub fn add<T: CommandExt + crate::builder::ext::Extension>(mut self, tagged: T) -> Self {
+        self.app_ext.set(tagged);
```

Comment:
`app_ext` is different than a `command_ext` and we need to carefully consider this.

A `Command` can either be a top-level / application command or a subcommand.  `Command` builder methods deal with one, the other, or both.  `app_ext` was created for dealing with top-level / application-related data. 

Even if we add a new `ext` field, we'd need to decide how to name the methods related to it.